magellan-legacy/src/main/java/com/wealthfront/magellan/ScreenGroup.java

```diff
@@ -37,9 +37,9 @@ public void addScreens(List<S> screens) {
 
   @Override
   protected void onShow(Context context) {
+    super.onShow(context);
```

Comment:
Isn't `ScreenGroup` obsolete if `Screen`s are `Navigable`? Should we just make this an empty class?

Reply:
Changed screen to only add screen and attach them to the screengroups lifecycle.